app/src/main/java/de/danoeh/antennapod/adapter/AllEpisodesListAdapter.java

```diff
@@ -125,6 +128,22 @@ public View getView(int position, View convertView, ViewGroup parent) {
                 holder.progress.setVisibility(View.GONE);
             }
 
+            TypedArray typeDrawables = context.obtainStyledAttributes(
+                    new int[]{R.attr.type_audio, R.attr.type_video});
+            final int[] labels = new int[]{R.string.media_type_audio_label, R.string.media_type_video_label};
+            MediaType mediaType = item.getMedia().getMediaType();
```

Comment:
Not every item has media so you have to check if getMedia() returns null before you do anything else.

I see that we're already in a conditional block that has done this check, so just use the existing 'media' variable so that it doesn't trip anyone up in the future.